packages/php-wasm/compile/Dockerfile

```diff
@@ -522,7 +522,7 @@ COPY ./build-assets/php_wasm.c /root/
 
 RUN if [ "$EMSCRIPTEN_ENVIRONMENT" = "node" ]; then \
         # Add nodefs when building for node.js
-        echo -n ' -lnoderawfs.js -s NODERAWFS=1 ' >> /root/.emcc-php-wasm-flags; \
+        echo -n ' -lnodefs.js ' >> /root/.emcc-php-wasm-flags; \
```

Comment:
@adamziel Should this be a flag?

Reply:
If `php.mount({ root: '/' }, '/')` works, then we can remove noderawfs entirely

Reply:
It doesn't work, but every top-level directory can be mounted so that's pretty close.